src/rpc/rawtransaction.cpp

```diff
@@ -28,6 +28,7 @@
 #include <script/standard.h>
 #include <uint256.h>
 #include <util/bip32.h>
+#include <util/moneystr.h>
```

Comment:
in commit 9c6a8b52394c9fdbb8c2296a77614b1c0114db49

Why the include? I can't see what it is for

Reply:
It was in the wrong commit. It's used for `FormatMoney` in RPC helps.